diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+Graces-MacBook-Pro
```

Comment:
Graces-MacBook-Pro is the name of the mac. The directory you would be in is /Users/grace/wdi/try